### ISSUE TITLE
amenity=music_school not rendered

### ISSUE THREAD

**Issue by MarcMart** [gravitystorm/openstreetmap-carto#4411]
### Expected behavior
Music schools are a very common amenity world wide which I believe justifies renderization with a proper icon or at least the standard dot. 

### Actual behavior
At this moment there is no icon, not even a dot. Only in case the address is added to the POI the house number is shown

### Links and screenshots illustrating the problem
Example of music school in JOSM
![image](https://user-images.githubusercontent.com/58865443/119101527-623c4d80-ba19-11eb-8784-ae4ba1fd03a5.png)

Rendering in standard Carto on openstreetmap.org:
![image](https://user-images.githubusercontent.com/58865443/119101694-8ef06500-ba19-11eb-8498-de534d7afeca.png)

Thanks for taking a look at this!

**Comment by Discostu36** [comment]
I agree. The tag is well established with over 5,000 uses worldwide and [documented on the wiki](https://wiki.openstreetmap.org/wiki/Tag:amenity%3Dmusic_school).

**Comment by polarbearing** [comment]
We currently render driving schools with a purple shop dot and a label. 
Maybe we could add the music school to this approach, until we find a nicer solution.

**Comment by MarcMart** [comment]
I agree. Could we also include amenity=language_school? I see the same happens here. Purple dot and label should be the minimum until we find a suitable icon.
This is the first time I open an issue here. What is the process for the creation/acceptance(implementation of an icon?

**Comment by HolgerJeromin** [comment]
Someone has to create an icon ([guidelines](https://github.com/gravitystorm/openstreetmap-carto/blob/master/CONTRIBUTING.md#map-icon-guidelines)) and someone has to do the "coding" and testing.
Best done after an general OK to the idea from an maintainer to not waste time.

**Comment by MarcMart** [comment]
Thanks for the quick reply. How can a general ok to the idea from maintainer be achieved?

**Comment by imagico** [comment]
There does not seem to be a well established practice in tagging of music schools in OSM at the moment.  In particular traditional conservatories (conservatorio in Italian) are predominantly not mapped with this tag.  There seem to be many mappers who prefer to tag this as a subtype of amenity=school/college/university rather than a separate thematic class of amenity which engrosses anything from a single person business providing guitar lessons to a university level music academy.

**Comment by polarbearing** [comment]
@imagico - not clear to me what you mean. 
amenity=music_school cannot be used as a subtag of amenity=school/college/university simply because they collide on the key. 
Note that the [wiki](https://wiki.openstreetmap.org/wiki/Tag:amenity%3Dmusic_school) explicitly declares to use: _"amenity=college - for an institute of Further Education which offers a certification or diploma in music"_.
The perspective that this style should only render tags that are used with 100% consistency blocks a couple of requests already, and is impossible to achieve.

**Comment by MarcMart** [comment]
I am very much in favor of consistent tagging. As Discostu36 has pointed out amenity=music_school has a wiki entry and is being used world wide over 5000 times - actually when searching taginfo for "music" alone it is second on the list, right after shop=music. What would the alternative tagging with amenity=college/university etc look like?

**Comment by imagico** [comment]
@polarbearing - you have to be more specific with what you don't understand.

For generally better understanding.  We here in OSM-Carto look primarily at the actual use of tags in the OSM database.  The documentation on the wiki is only of interest for us as far as it accurately describes this use.  In this case i looked at the tagging of conservatories in different parts of Europe and none of them was tagged `amenity=music_school`.

If mappers change this and decide on unified tagging for - as mentioned - anything from a single person business providing guitar lessons to a university level music academy that would change things.  Just as it would if a consensus develops to delineate `amenity=music_school` towards music schools that are tagged amenity=school/college/university - like the mentioned conservatories.  But at the moment the only thing that seems clear is that there is no consensus on tagging in this domain.

**Comment by Janjko** [comment]
> i looked at the tagging of conservatories in different parts of Europe

The reason they chose `amenity=school` is because `amenity=music_school` isn't rendered. Those institutions are very important to those mappers, so they tagged them wrongly because at some point, when important institutions are in question, everybody tags for the renderer. So that point isn't very strong. Actually, that point is against not rendering.

And also, a lot of big music schools teach other classes, not just music. So in those cases, mappers might choose to tag them as a `amenity=school` because of that.

As for "from conservatory to one man teaching guitar", isn't `amenity=school` the same? In my country there are schools with one teacher and one pupil in several smaller villages, and there are schools with a thousand students. Tagged exactly the same. So that point also isn't valid.

At this point there are more than 10 000 `amenity=music_school` elements in OSM, more than `shop=music`. I think there is no reason for not rendering these.

**Comment by c-h-r-i-s-t-i-a-n** [comment]
In the mean time we have a usage count of 13.000. 
Do we need a nice icon?

**Comment by aceman444** [comment]
There are many music schools that teach kids playing some instrument in evening hours, and they are no real schools. 
Why do you insist music_school should also be used on conservatoriums, which are full-time educational institutions similar to a college or university? Or why is it a problem that conservatoriums are not tagged amenity=music_school?

**Comment by dch0ph** [comment]
> There are many music schools that teach kids playing some instrument in evening hours, and they are no real schools.

We don't generally map/render small businesses run from private houses. These aren't "educational institutions".

It would be useful to survey how `amenity=music_school` was being used and check that they aren't largely in residential areas.

**Comment by imagico** [comment]
> It would be useful to survey how amenity=music_school was being used and check that they aren't largely in residential areas.

More generally speaking a renewed in depth look at mapping practice is required here.

Back in 2021 it was clear that there was no consensus on the semantic scope of `amenity=music_school`.  If this has changed there are the following possibilities:

1. `amenity=music_school` is limited to providers of informal music education (like guitar/piano lessons by individuals) and is the consensus tag for that.
2. `amenity=music_school` is covering any and all places where schooling of some form related to music happens - from individuals providing practical lessons to large academic institutions and is the consensus tag across this range.  This is what the wiki seems to indicate to me.
3. Something in between the two.  This would require identifying (and preferably documenting) where the semantic delineation is between facilities tagged `amenity=music_school` and such that are tagged differently.

What the best way is to depict `amenity=music_school` (and if to do so is a good idea under our goals) will depend on this so my suggestion for how to proceed here is to 

* analyze current use of the tag.
* present your findings for discussion here.
* develop a design concept to depict music education in OSM-Carto that reflects the identified consensus tagging practice and discuss it here.
* write a PR implementing that.

> Why do you insist music_school should also be used on conservatoriums [...]

Please don't make false claims on what others have said.  If you want to argue against what has been said here please identify the statement you want to challenge and present your argument.

**Comment by aceman444** [comment]
> There does not seem to be a well established practice in tagging of music schools in OSM at the moment. In particular traditional conservatories (conservatorio in Italian) are predominantly not mapped with this tag.

I refer to 2 statements like this. It states the tag is not used for conservatories somehow implying it is a problem.
If you didn't imply this, then what did you mean?

Yes, the wiki says conservatories also count under music_school. But maybe people want to distinguish private/state operated institutions that provide after-school music courses from universities/colleges that teach music full-time (a conservatorium), so they use different tags (music_school vs. college).

It is still unclear how the usage affects rendering. How will you render it differently if music_school applies to conservatoriums too or when it only means music courses in not-really-schools (just called so in many countries).
